CMakeLists.txt

```diff
@@ -100,3 +100,41 @@ else()
         endif()
     endif()
 endif()
+
+#Added Qt6 support via QT_ANDROID_EXTRA_LIBS
+# With the recent update and the introduction of the QT_ANDROID_EXTRA_LIBS variable, this modification is necessary to copy the openssl files to the required path in Qt6.
+# https://doc.qt.io/qt-6/cmake-target-property-qt-android-extra-libs.html
+if(QT_VERSION GREATER 6.0.0)
+    set_property(TARGET ${PROJECT_NAME} PROPERTY QT_ANDROID_EXTRA_LIBS
```

Comment:
"APPEND PROPERTY" instead of "PROPERTY" to avoid overriding the existing libs list (if any), though, this is also pre-existing in current code

Reply:
Yes!
Done, thanks for this suggestion.